lib/manager/common.ts

```diff
@@ -26,6 +26,15 @@ export interface ExtractConfig extends ManagerConfig {
   versioning?: string;
 }
 
+export interface CustomExtractConfig extends ExtractConfig {
+  matchStrings: string[];
+  depNameTemplate?: string;
+  lookupNameTemplate?: string;
+  currentValueTemplate?: string;
+  datasourceTemplate?: string;
+  versioningTemplate?: string;
+}
+
```

Comment:
Can we reuse the  `CustomManager` interface above? We can move it to the `lib/types` path.
```suggestion
export type CustomExtractConfig = CustomManager & ExtractConfig;
```